### PR TITLE
Visitor can see datapoints with correct colors - FRONT END

### PR DESCRIPTION
**Pull request by Carrosen**
# [PT Story](https://www.pivotaltracker.com/story/show/166590146)

## Changes proposed in this PR
* Six different colors of datapoints depending on status, category and visitor/user
* The colors are implemented in Map, AdminPage, EntryPopup and AdminPopup
* Styles application to match new colors

## What we have learned
* Nested conditionals

## Screenshots
Screenshot from 2019-06-26 10-34-23

Screenshot from 2019-06-26 10-38-07

Screenshot from 2019-06-26 10-35-07

Screenshot from 2019-06-26 10-37-46

### CODE REVIEW

File: src/reduxTokenAuthConfig.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const loc = 'http://localhost:3002/api/v1/auth'
 const her = 'https://being-stockholm.herokuapp.com/api/v1/auth'
 
 const config = {
-  authUrl: her,
+  authUrl: loc,
   userAttributes: {
```
Comment (raptorf1) [inline src/reduxTokenAuthConfig.js]: back to `her`

File: src/index.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ setupInterceptors()
 const loc = 'http://localhost:3002'
 const her = 'https://being-stockholm.herokuapp.com'
 
-axios.defaults.baseURL = her
+axios.defaults.baseURL = loc
```
Comment (raptorf1) [inline src/index.js]: back to `her`

**Review by raptorf1 — CHANGES_REQUESTED** [review]

File: src/reduxTokenAuthConfig.js (hunk 1 above)
Comment (Carrosen) [reply src/reduxTokenAuthConfig.js]: yes

File: src/index.js (hunk 2 above)
Comment (Carrosen) [reply src/index.js]: yes

**Pull request merged**